code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -468,12 +468,14 @@
 
 /obj/structure/closet/attack_hand_secondary(mob/user, modifiers)
 	if(!user.canUseTopic(src, BE_CLOSE) || !isturf(loc))
-		return
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
 
 	if(!opened && secure)
 		togglelock(user)
 		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
 
+	return SECONDARY_ATTACK_CALL_NORMAL
```

Comment:
Don't return CALL_NORMAL, it's only there until everything is on secondary procs.

Reply:
Doneified.